modules/AOS_PDF_Templates/generatePdf.php

```diff
@@ -139,28 +138,40 @@ function ($matches) {
 
 $printable = str_replace("\n", "<br />", $converted);
 
-if ($task == 'pdf' || $task == 'emailpdf') {
+if ($task === 'pdf' || $task === 'emailpdf') {
     $file_name = $mod_strings['LBL_PDF_NAME'] . "_" . str_replace(" ", "_", $bean->name) . ".pdf";
 
-    ob_clean();
     try {
-        $orientation = ($template->orientation == "Landscape") ? "-L" : "";
-        $pdf = new mPDF('en', $template->page_size . $orientation, '', 'DejaVuSansCondensed', $template->margin_left, $template->margin_right, $template->margin_top, $template->margin_bottom, $template->margin_header, $template->margin_footer);
-        $pdf->SetAutoFont();
-        $pdf->SetHTMLHeader($header);
-        $pdf->SetHTMLFooter($footer);
-        $pdf->WriteHTML($printable);
-        if ($task == 'pdf') {
-            $pdf->Output($file_name, "D");
+        $orientation = ($template->orientation === "Landscape") ? "-L" : "";
```

Comment:
Hi @Dillon-Brown, was wondering if it would be best to pass a standard `orientation`, like `landscape` or `portrait` instead of `-L` joined on the `format`. Then each pdf engine implementation would now know to set it, as it may differ from one to another.

Reply:
Agreed, looking at mPDF it looks like it accepts orientation in ```Landscape``` which is what we generate with the template vars anyway.

Pushed along with a small PHP notice fix in the AOR_Reports controller when trying to count null:

```php
        if (is_countable($graphs)) {
            $countOfCharts = count($graphs);
        }
```

Reply:
thank you